packages/livechat/src/lib/transcript.ts

```diff
@@ -23,12 +33,12 @@ const promptTranscript = async () => {
 		text: message,
 	}).then((result) => {
 		if (typeof result.success === 'boolean' && result.success) {
-			return Livechat.requestTranscript(email, { token, rid: _id });
+			return Livechat.requestTranscript(email, { rid: _id });
```

Comment:
Why did we remove `token` from here? Was it expected?

Reply:
The new SDK does not requires tokens to be sent, it is stored by the SDK internally